strr-examiner-web/tests/unit/dashboard.spec.ts

```diff
@@ -94,4 +99,21 @@ describe('Examiner Dashboard Page', () => {
     expect(applicationText).toContain(displayContactFullName(registration.primaryContact))
     expect(applicationText).toContain(registration.unitAddress.city)
   })
+
+  it('renders the assignee column correctly', () => {
+    expect(wrapper.findTestId('applications-table').exists()).toBe(true)
+    const { applications } = wrapper.vm.applicationListResp.value
+    const appWithReviewer = applications.find(app =>
+      app.adjudicator === mockHostApplicationWithReviewer.header.reviewer?.username
+    )
+    expect(appWithReviewer).toBeDefined()
+    expect(appWithReviewer?.adjudicator).toBe(mockHostApplicationWithReviewer.header.reviewer?.username)
```

Comment:
Optional chaining operator on the `appWithReviewer` is redundant here since it is checked on the line before.

Reply:
Done